sys/key_rev_value.js

```diff
@@ -30,17 +30,20 @@ KRVBucket.prototype.getBucketInfo = function(restbase, req, options) {
 };
 
 KRVBucket.prototype.makeSchema = function(opts) {
-    var schemaVersionMajor = 1;
+    var schemaVersionMajor = 2;
 
     var schema =  {
+        // 1000 external version bumps per internal version bump ought to be
+        // enough for anybody.
+        version: schemaVersionMajor * 1000 + (opts.version || 0),
         options: {
-            compression: [
-                {
-                    algorithm: 'deflate',
-                    block_size: 256
-                }
-            ]
+            compression: opts.compression,
+            updates: opts.updates || {
+                pattern: 'timeseries'
+            },
         },
+        revisionRetentionPolicy: opts.revisionRetentionPolicy
+            || opts.retention_policy,
```

Comment:
@gwicke we are providing a latest-1 rev policy in a key_value storage, this case doesn't seem different